test_fms/fms2_io/atmosphere_restart_file_test.inc

```diff
@@ -51,6 +51,8 @@ subroutine atmosphere_restart_file(domain, nz, nt, debug_flag)
   character(len=256) :: att
   character(len=256), dimension(:), allocatable :: buffer
   character(len=256), dimension(:), allocatable :: buffer2
+  character(len=256) :: str_buffer
+  character(len=256) :: str_buffer2
```

Comment:
These variables should have doxygen comments stating what they are

Reply:
it's in the name. it's a string buffer.

Reply:
Is there markdown for an eye roll?